.github/workflows/PR_info_check.yml

```
@@ -6,12 +6,33 @@ on:
       - edited
       - reopened
       - synchronize
+    branches:
+      - develop
+      - master
 
 jobs:
   check-commit-message:
     name: Check Commit Message
     runs-on: ubuntu-latest
     steps:
+      - name: Check PR JIRA reference
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(\[WIP\] )?((EPMLSTRCMW-\d{1,4})|(IS-00)) .*$'
```

maybe
```suggestion
          pattern: '^(\[WIP\] )?((EPMLSTRCMW-\d+)|(IS-00)) .*$'
```
to not care at all about task numbers?

---

Done